sdk/storage/Azure.Storage.Files/README.md

```diff
@@ -9,79 +9,178 @@ Windows, Linux, and macOS. Additionally, Azure file shares can be cached on
 Windows Servers with Azure File Sync for fast access near where the data is
 being used.
 
-[Source code][source] | [Package (NuGet)][package] | [API reference documentation][rest_docs] | [Product documentation][product_docs]
+[Source code][source] | [Package (NuGet)][package] | [API reference documentation][docs] | [REST API documentation][rest_docs] | [Product documentation][product_docs]
 
 ## Getting started
+
 ### Install the package
+
 Install the Azure Storage Files client library for .NET with [NuGet][nuget]:
 
 ```Powershell
-Install-Package Azure.Storage.Files
+dotnet add package Azure.Storage.Files --version 12.0.0-preview.1
 ```
 
-**Prerequisites**: You must have an [Azure subscription][azure_sub], and a
+### Prerequisites
+
+You need an [Azure subscription][azure_sub] and a
 [Storage Account][storage_account_docs] to use this package.
 
-To create a Storage Account, you can use the [Azure Portal][storage_account_create_portal],
-[Azure PowerShell][storage_account_create_ps] or [Azure CLI][storage_account_create_cli]:
+To create a new Storage Account, you can use the [Azure Portal][storage_account_create_portal],
+[Azure PowerShell][storage_account_create_ps], or the [Azure CLI][storage_account_create_cli].
+Here's an example using the Azure CLI:
+
+```Powershell

```

Comment:
nit:  This appears to be the Azure CLI, not PowerShell

Reply:
You can run the Azure CLI from within PS just fine.